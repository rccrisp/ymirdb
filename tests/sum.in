set a 1 2 3 4
sum a
set b a 1
set c b 1
set d c 1
set e d -1
sum b
sum c
sum d
sum e
set a 0
sum e
pop a
sum e
bye
