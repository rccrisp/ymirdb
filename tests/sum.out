> ok

> 10

> ok

> ok

> ok

> ok

> 11

> 12

> 13

> 12

> ok

> 2

> 0

> 2

> bye
